crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -2961,9 +2962,264 @@ TEST_P(PBKDF2_ServiceIndicatorTest, PBKDF2) {
+    hmac_sha1_iv, sizeof(hmac_sha1_iv),
+    true,
+    hmac_sha1_ko, sizeof(hmac_sha1_ko),
+    AWSLC_APPROVED
+  },
+  {
+    EVP_sha224,
+    hmac_sha224_ki, sizeof(hmac_sha224_ki),
+    hmac_sha224_fixedinput, sizeof(hmac_sha224_fixedinput),
+    nullptr, 0,  // included in fixedinput
+    hmac_sha224_iv, sizeof(hmac_sha224_iv),
+    true,
+    hmac_sha224_ko, sizeof(hmac_sha224_ko),
+    AWSLC_APPROVED
+  },
+  {
+    EVP_sha256,
+    hmac_sha256_ki, sizeof(hmac_sha256_ki),
+    hmac_sha256_fixedinput, sizeof(hmac_sha256_fixedinput),
+    nullptr, 0,  // included in fixedinput
+    hmac_sha256_iv, sizeof(hmac_sha256_iv),
+    true,
+    hmac_sha256_ko, sizeof(hmac_sha256_ko),
+    AWSLC_APPROVED
+  },
+  {
+    EVP_sha384,
+    hmac_sha384_ki, sizeof(hmac_sha384_ki),
+    hmac_sha384_fixedinput, sizeof(hmac_sha384_fixedinput),
+    nullptr, 0,  // included in fixedinput
+    hmac_sha384_iv, sizeof(hmac_sha384_iv),
+    true,
+    hmac_sha384_ko, sizeof(hmac_sha384_ko),
+    AWSLC_APPROVED
+  },
+  {
+    EVP_sha512,
+    hmac_sha512_ki, sizeof(hmac_sha512_ki),
+    hmac_sha512_fixedinput, sizeof(hmac_sha512_fixedinput),
+    nullptr, 0,  // included in fixedinput
+    hmac_sha512_iv, sizeof(hmac_sha512_iv),
+    true,
+    hmac_sha512_ko, sizeof(hmac_sha512_ko),
+    AWSLC_APPROVED
+  },
+  {
+    EVP_md5,
+    hmac_sha1_ki, sizeof(hmac_sha1_ki),
+    hmac_sha1_fixedinput, sizeof(hmac_sha1_fixedinput),
+    nullptr, 0,  // included in fixedinput
+    hmac_sha1_iv, sizeof(hmac_sha1_iv),
+    true,
+    hmac_sha1_ko, sizeof(hmac_sha1_ko),
+    AWSLC_NOT_APPROVED
+  },
+};
+
+class KBKDFTestSI : public testing::TestWithParam<KBKDFTestVectorSI> {
+};
+
+TEST_P(KBKDFTestSI, KBKDFCAVPVectorsSI) {
+  const KBKDFTestVectorSI &test = GetParam();
+  FIPSStatus approved = AWSLC_NOT_APPROVED;
+
+  uint8_t *output = new uint8_t[test.expected_len];
+  ASSERT_NE(output, nullptr);
+
+  CALL_SERVICE_AND_CHECK_APPROVED(approved,
+    ASSERT_TRUE(KBKDF_feedback(output, test.expected_len,
+                               test.md_func(),
+                               test.ki, test.ki_len,
+                               test.label, test.label_len,
+                               test.context, test.context_len,
+                               test.iv, test.iv_len,
+                               test.use_counter)));
+  if (test.expect_approved) {
+    EXPECT_EQ(Bytes(test.expected, test.expected_len),
+              Bytes(output, test.expected_len));
+  }
+  EXPECT_EQ(approved, test.expect_approved);
+
+  delete[] output;
+}
+
+INSTANTIATE_TEST_SUITE_P(KBKDF_ServiceIndicatorTest, KBKDFTestSI,
+                         testing::ValuesIn(nonzero_iv_vectors));
+
 // SSHKDF test vector from NIST CAVS 14.1 test vectors
 // https://csrc.nist.gov/Projects/Cryptographic-Algorithm-Validation-Program/Component-Testing#KDF135
-static const char kSSHKDFtype = EVP_KDF_SSHKDF_TYPE_INITIAL_IV_CLI_TO_SRV;
+const char kSSHKDFtype = EVP_KDF_SSHKDF_TYPE_INITIAL_IV_CLI_TO_SRV;
```

Comment:
Any reason for all these static changes?

Reply:
Just making them consistent with the other constants in the file.